client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -1,97 +1,206 @@
+import { untrailingslashit } from 'lib/route';
 import Button from 'components/button';
-import Card from 'components/card';
 import getJetpackProductInstallProgress from 'state/selectors/get-jetpack-product-install-progress';
 import JetpackProductInstall from 'my-sites/plans/current-plan/jetpack-product-install';
 import ProgressBar from 'components/progress-bar';
-import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import ThankYouCard from './thank-you-card';
+
+const INSTALL_STATE_COMPLETE = 1;
+const INSTALL_STATE_UNCOMPLETE = 2;
 
 export class PaidPlanThankYouCard extends Component {
-	getMyPlanRoute() {
-		const { siteSlug } = this.props;
+	render() {
+		const {
+			fileModDisableReason,
+			hasMinimumJetpackVersion,
+			installProgress,
+			installState,
+			isSiteMainNetworkSite,
+			isSiteMultiSite,
+			site,
+			translate,
+		} = this.props;
 
-		return `/plans/my-plan/${ siteSlug }`;
-	}
+		const securityIllustration = '/calypso/images/illustrations/security.svg';
+		const fireworksIllustration = '/calypso/images/illustrations/fireworks.svg';
 
-	render() {
-		const { progressComplete, translate } = this.props;
+		// Jetpack is too old
+		if ( ! hasMinimumJetpackVersion ) {
+			// Link to "Plugins" page in wp-admin
+			let wpAdminPluginsUrl = get( site, 'options.admin_url' );
+			wpAdminPluginsUrl = wpAdminPluginsUrl
+				? untrailingslashit( parseUrl( wpAdminPluginsUrl ).pathname ) + '/plugins.php'
+				: undefined;
+
+			return (
+				<ThankYouCard
+					illustration={ fireworksIllustration }
+					showContinueButton={ ! wpAdminPluginsUrl }
+					showHideMessage={ wpAdminPluginsUrl }
+					title={ translate( 'Thank you for your purchase!' ) }
+				>
+					<p>
+						{ preventWidows(
+							translate(
+								'Unfortunately, we are unable to set up your plan because your site has an older version of Jetpack. Please upgrade Jetpack.'
+							)
+						) }
+					</p>
+					{ wpAdminPluginsUrl && (
+						<p>
+							<Button primary href={ wpAdminPluginsUrl } target="_blank">
+								<span>{ translate( 'Upgrade Jetpack' ) }</span>
+								<Gridicon icon="external" />
+							</Button>
+						</p>
+					) }
+				</ThankYouCard>
+			);
+		}
+
+		// We cannot install anything for this site
+		if ( fileModDisableReason && fileModDisableReason.length > 0 ) {
+			return (
+				<ThankYouCard
+					illustration={ fireworksIllustration }
+					showHideMessage
+					title={ translate( 'Thank you for your purchase!' ) }
+				>
+					<p>
+						{ preventWidows(
+							translate(
+								"Unfortunately, we can't modify files on your site, so you'll need to set up your plan features manually."
+							)
+						) }
+					</p>
+					<p>
+						{ preventWidows(
+							translate( "Don't worry. We'll quickly guide you through the setup process." )
+						) }
+					</p>
+					<p>
+						<Button primary href={ SETTING_UP_PREMIUM_SERVICES } target="_blank">
+							<span>{ translate( 'Set up features' ) }</span>
```

Comment:
To add space, you'll probably want:
```suggestion
							<span>{ translate( 'Set up features' ) }</span> <Gridicon icon="external" />
```

I believe prettier will format that as:

```suggestion
							<span>{ translate( 'Set up features' ) }</span>
							{ ' ' }
							<Gridicon icon="external" />
```

Reply:
Nice and simple! :tada: